lib/sky-pages-module-generator.js

```diff
@@ -19,18 +33,19 @@ function getSource(skyAppConfig) {
   const cssFilePaths = get(skyAppConfig, 'skyux.app.styles', []);
 
   // Add the default theme style sheet only if the consumer hasn't provided it.
-  const defaultThemeStyleSheet = '@skyux/theme/css/sky.css';
   if (cssFilePaths.indexOf(defaultThemeStyleSheet) === -1) {
     cssFilePaths.unshift(defaultThemeStyleSheet);
   }
 
+  insertThemePaths(skyAppConfig, cssFilePaths);
+
   let styles = '';
   let hasExternal = false;
   cssFilePaths.forEach((filePath) => {
     if (/^http/.test(filePath)) {
       hasExternal = true;
     } else {
-      styles += `require('!style-loader!css-loader!sass-loader!${filePath}');\n`;
```

Comment:
When I backported this code to version 3, I changed the loader prefix to back `style-loader`. Should it be changed back to `!style-loader!css-loader!sass-loader` for the RC?

Reply:
Hm, yeah probably. Good catch!